AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/ModelValueConverter+SQLite.swift

```diff
@@ -0,0 +1,94 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import Foundation
+import SQLite
+
+private extension Bool {
+
+    var intValue: Int {
+        return self ? Int(1) : Int(0)
+    }
+
+}
+
+public struct SQLiteModelValueConverter: ModelValueConverter {
+
+    public typealias SourceType = Any?
+    public typealias TargetType = Binding?
+
+    public func convertToTarget(from source: Any?, fieldType: ModelFieldType) throws -> Binding? {
```

Comment:
Since these don't rely on any internal state, consider making them static methods, to avoid an unnecessary instantiation at the call site

Reply:
good call. I initially had an internal state but then refactored it to get rid of the state and didn't change it to static. Will do.